bahmni_product/models/product.py

```diff
@@ -138,43 +62,9 @@ def name_get(self):
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
     
-    def _compute_quantities(self):
```

Comment:
Is this not being used anywhere, and hence deleted?

Reply:
this logic are covered in the odoo16 itself.so we removed this code.